packages/tools/piw-utils-internal/src/utils/StylingPropertyUtils.ts

```diff
@@ -1,5 +1,15 @@
 import { CSSProperties } from "react";
-import { Dimensions } from "../../typings/shared";
+
+export type WidthUnitEnum = "percentage" | "pixels";
+
+export type HeightUnitEnum = "percentageOfWidth" | "pixels" | "percentageOfParent";
+
+export interface Dimensions {
+    widthUnit: WidthUnitEnum;
+    width: number;
+    heightUnit: HeightUnitEnum;
+    height: number;
+}
```

Comment:
for native: I extracted all of this code into `piw-internal` since we're reusing it at multiple places right now

Reply:
👍🏻